configuration/pih/pih-config-haiti-cange.json

```diff
@@ -26,9 +26,10 @@
     "socioEconomics",
     "systemAdministration",
     "todaysVisits",
+    "uhmVitals",
```

Comment:
What's the difference between the components:  uhmVitals and vitals?  Why does cange have uhmVitals but not other health facilities (belladere, boucan carre)?